WordPress/src/test/java/org/wordpress/android/util/FormattableContentUtilsTest.kt

```diff
@@ -0,0 +1,338 @@
+package org.wordpress.android.util
+
+import com.nhaarman.mockito_kotlin.whenever
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.fluxc.tools.FormattableContent
+import org.wordpress.android.fluxc.tools.FormattableMedia
+import org.wordpress.android.fluxc.tools.FormattableMeta
+import org.wordpress.android.fluxc.tools.FormattableRange
+
+@RunWith(MockitoJUnitRunner::class)
+class FormattableContentUtilsTest {
+    @Mock
+    lateinit var content: FormattableContent
```

Comment:
You shouldn't be mocking basic model objects. I'd build an actual object instead with real fields. That goes for this whole file.

Reply:
👍